packages/ui-components/src/components/Chart/Table/Table.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import MuiTableContainer from '@material-ui/core/TableContainer';
+import { DataTable } from '@tupaia/ui-components';
```

Comment:
From what I can see, there isn't a `DataTable` export in ui-components (also should we just go through the relative directory rather than the named library, or does that cause issues with the code splitting?)